src/components/inputsList/InputsList.test.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import InputsList from './InputsList';
+
+it('renders inputs List', () => {
```

Comment:
1) this test is pointless, it tests nothing. test should follow this rules:
* **Arrange** all necessary preconditions and inputs.
* **Act** on the object or method under test.
* **Assert** that the expected results have occurred.

2) instead of ```javascript const div = document.createElement('div'); ``` and 
``` javascript 
ReactDOM.render(
         <InputsList inputs={inputs}/>,
        div);
```
use enzyme